pkg/network/state.go

```diff
@@ -342,11 +350,10 @@ func (ns *networkState) StoreClosedConnections(closed []ConnectionStats) {
 func (ns *networkState) storeClosedConnections(conns []ConnectionStats) {
 	for _, client := range ns.clients {
 		for _, c := range conns {
-			key := c.ByteKeyNAT(ns.buf)
+			key := string(c.ByteKey(ns.buf))
```

Comment:
I'm wondering if this will result in a regression (HTTP orphan aggregations)– we should probably validate this in the load test environment)
Is this "conceptually" equivalent in the sense that we won't merge apples and oranges because two connections with same ByteKey but different NAT translations would have distinct cookies?

(We should probably remove `ByteKeyNAT` too, no? I don't think it's being used anywhere else)